portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -30,15 +35,29 @@
 import com.google.appengine.api.datastore.Query;
 import com.google.appengine.api.datastore.Query.SortDirection;
 import javax.servlet.annotation.WebServlet;
+
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+import com.google.api.client.googleapis.auth.oauth2.GoogleIdToken;
+import com.google.api.client.googleapis.auth.oauth2.GoogleIdToken.Payload;
+import com.google.api.client.googleapis.auth.oauth2.GoogleIdTokenVerifier;
+
+import com.google.api.client.http.HttpTransport;
+import com.google.api.client.http.javanet.NetHttpTransport;
+import com.google.api.client.json.jackson2.JacksonFactory;
+
 /** Servlet that returns comments on portfolio */
 @WebServlet("/comments")
 public class DataServlet extends HttpServlet {
   private int commentLimit = 25; //default set to 25
```

Comment:
Also missed this: I would move this line below the constants declarations, with a new line in between.
https://google.github.io/styleguide/javaguide.html#s3.4.2-ordering-class-contents doesn't specify but that's my recommendation.